website/versioned_docs/version-0.11/plugins/joblib.md

```diff
@@ -0,0 +1,81 @@
+---
+id: joblib
```

Comment:
I am a bit torn about how much duplication we have between the readme and the website.
Maybe we should just change the README to point to the website?

Reply:
Yes, I'd prefer that as well